pkgs/development/compilers/graalvm/community-edition/graalpy/hashes.nix

```diff
@@ -0,0 +1,22 @@
+# Generated by pkgs/development/compilers/graalvm/community-edition/update.sh script
+{
+  "version" = "23.1.0";
+  "graalpy" = {
+    "aarch64-linux" = {
+      sha256 = "0n0zz86h7jsqgdiyj6vj7qw57ny40jpmfvylyxq70riy86a4zp67";
+      url = "https://github.com/oracle/graalpython/releases/download/graal-23.1.0/graalpy-community-23.1.0-linux-aarch64.tar.gz";
```

Comment:
I am using the non-JVM variants here, e.g.: `graalpy-community-jvm`. Those versions includes the libraries to interface with JVM, that I imagine enables usage of mixed languages with interop (https://www.graalvm.org/latest/reference-manual/js/JavaInteroperability/).

Not sure who is using this though, and this complicates the packaging, so for now I am going to package the standalone versions instead, and if there is someone that needs those libraries please open an issue.

Reply:
I'm a bit late to this, but I thought I would add:

For Truffle polyglot projects, if I was going to write one, I would be using something like `mvn2nix` and I think I would be able to fetch everything necessary from Maven instead of needing them in nixpkgs.

(So, packaging only the standalone versions seems fine.)